e2e-testing/__tests__/transfers.test.ts

```diff
@@ -0,0 +1,137 @@
+import Long from 'long';
+import {
+  BECH32_PREFIX,
+  IndexerClient,
+  LocalWallet,
+  Network,
+  SocketClient,
+  SubaccountInfo,
+  ValidatorClient,
+} from '@dydxprotocol/v4-client-js';
+import {
+  Ordering,
+  SubaccountTable,
+  TransferColumns,
+  TransferFromDatabase,
+  TransferTable,
+} from '@dydxprotocol-indexer/postgres';
+import * as utils from './helpers/utils';
+import { DYDX_LOCAL_ADDRESS, DYDX_LOCAL_MNEMONIC } from './helpers/constants';
+
+describe('transfers', () => {
```

Comment:
nit: Do we need to do any beforeAll, beforeEach, afterAll, and afterEach setups?

Reply:
yeah, I was thinking about the best way to do this.
I think we may need to add this in the future depending on tests we write, but for now I'm going to omit it from this PR.